packages/automated-dispute/src/types/epochs.ts

```diff
@@ -0,0 +1,14 @@
+import { Timestamp } from "@ebo-agent/shared";
+
+/**
+ * Type representing an epoch's data.
+ *
+ * @property {bigint} epoch - epoch number
+ * @property {bigint} epochFirstBlockNumber - number of the first block of the epoch
+ * @property {Timestamp} epochStartTimestamp - timestamp of the first block of the epoch
+ */
+export type Epoch = {
+    epoch: bigint;
+    epochFirstBlockNumber: bigint;
+    epochStartTimestamp: Timestamp;
+};
```

Comment:
```suggestion
export type Epoch = {
    count: bigint;
    firstBlockNumber: bigint;
    startTimestamp: Timestamp;
};
```

What do you think if we do it this way. I noticed some places that u did Epoch["epoch"] and i think `Epoch["count"]` or `Epoch["number"]` might be more descriptive and cleaner.

Reply:
Ended up using `number`, on the basis of having `epoch.number` and also `block.number`. `count` feels a little bit more like there are elements inside the `Epoch`